plugins/ExtendedTools/objmgr.c

```diff
@@ -509,22 +537,23 @@ static BOOLEAN NTAPI EtEnumDirectoryObjectsCallback(
 static BOOLEAN NTAPI EtEnumCurrentDirectoryObjectsCallback(
     _In_ PPH_STRINGREF Name,
     _In_ PPH_STRINGREF TypeName,
-    _In_ POBJECT_CONTEXT Context
+    _In_ PET_OBJECT_CONTEXT Context
     )
 {
     if (PhEqualStringRef(TypeName, &DirectoryObjectType, TRUE))
     {
-        if (!EtTreeViewFindItem(Context->TreeViewHandle, Context->SelectedTreeItem, Name))
+        if (!EtTreeViewFindItem(Context->TreeViewHandle, Context->SelectedTreeItem, Name, FALSE))
         {
             EtTreeViewAddItem(Context->TreeViewHandle, Context->SelectedTreeItem, TRUE, Name);
         }
     }
     else
     {
-        INT index;
-        POBJECT_ENTRY entry;
+        PET_OBJECT_ENTRY entry;
 
-        entry = PhAllocateZero(sizeof(ET_OBJECT_ENTRY));
+        entry = PhCreateAlloc(sizeof(ET_OBJECT_ENTRY));
```

Comment:
🤔 - I see we're using `ObDereferenceObject` on this elsewhere? But we're not constructing the object, just using a raw allocation?